doc/blog/Wasm-JS series/Blog - WASM.md

```diff
@@ -0,0 +1,672 @@
+  SetAttribute("href", "#section2");
+  
+  // Set many attributes at once
+  SetAttribute(("target", "_blank"), ("referrerpolicy", "no-referrer"));
+  
+  // Remove attribute from DOM element
+  RemoveAttribute("href");
+  ```
+
+* The method `base.SetHtmlContent()` could be used to set arbitrary HTML content as child of the control.
+
+  ``` csharp
+  SetHtmlContent("<h2>Welcome on Uno Platform!</h2>");
+  ```
+
+  > Note: don't use this unless your control doesn't include any _managed_ children, or you'll get unexpected results.
+
+* Finally, it's possible to invoke an arbitrary bunch of JavaScript code by using the _static_ method `WebAssembleRuntime.InvokeJS()`. Once there, it directly in the browser, so it's possible to make almost anything. It's possible to use the `HtmlId` property of the element to locate it in JavaScript code.
+  If the control has been loaded (after de `Loaded` router event), it will be available directly by calling the JavaScript `document.getElementById()`. But it's also possible to access it before using the  `Uno.UI.WindowManager.current.getView(<HtmlId>)` function in JavaScript.
+
+To illustrate how it's possible to use this in a real application, let's create one to integrate a pretty simple _syntax highlighter_ named [`PrismJS`](https://prismjs.com/).
+
+# Sample - Integration of PrismJS in a project
+
+## 0. Before starting
+
+📝 To reproduce the code in this article, you must [prepare your development environment](https://platform.uno/docs/articles/get-started.html).
+
+## 1. Creating the projects
+
+🎯 This section is very similar to the [_Getting Started_ in the official documentation](https://platform.uno/docs/articles/getting-started-tutorial-1.html).
+
+1. Start **VisualStudio 2019**
+2. Click `Create a new project`
+   ![image-20200325113112235](image-20200325113112235.png)
+3. **Search for "Uno"** and pick `Cross-Platform App (Uno Platform)`. Pay attention to select the `App`, not the `Library` project type.
+   ![image-20200325113532758](image-20200325113532758.png)
+   Select it and click `Next`.
+4. Give a project name and folder as you wish. It will be named `PrismJsDemo` here.
+5. Click `Create` button.
+6. Delete the `.Droid`, `.iOS` and `.UWP` projects.
+   
+   > Note: it's definitely possible to build multi-platforms controls, but it's the goal of another article. So removing them will avoid misleading compilation errors for now.
+7. Right-click on the solution and pick `Manage NuGet Packages for Solution...`
+   ![image-20200325114155796](image-20200325114155796.png)
+8. Update to latest version of `Uno` dependencies. **DO NOT UPDATE THE `Microsoft.Extensions.Logging` dependencies** to latest versions.
+9. Press `CTRL-F5`. App should compile and start a browsing showing this:
+   ![image-20200325114609689](image-20200325114609689.png)
+   Note: if it's the first time you're using _Uno_, it could take some times to download the latest Mono-Wasm SDK into a _temp_ folder.
```

Comment:
```suggestion
   Note: if it's the first time you're using _Uno_, it could take some time to download the latest Mono-Wasm SDK into a _temp_ folder.
```